targets/mbedos5/jerryscript-mbed/jerryscript-mbed-drivers/source/setInterval-js.cpp

```diff
@@ -28,11 +28,20 @@ DECLARE_GLOBAL_FUNCTION(setInterval) {
     CHECK_ARGUMENT_TYPE_ALWAYS(global, setInterval, 0, function);
     CHECK_ARGUMENT_TYPE_ALWAYS(global, setInterval, 1, number);
 
-    jerry_acquire_value(args[0]);
     int interval = int(jerry_get_number_value(args[1]));
 
     int id = mbed::js::EventLoop::getInstance().getQueue().call_every(interval, jerry_call_function, args[0], jerry_create_null(), (jerry_value_t*)NULL, 0);
 
+    jerry_value_t result = jerry_set_property_by_index(this_obj, id, args[0]);
```

Comment:
But this will pollute the global object, won't it? Basically, it's OK with me to assign `args[0]` to a property _somewhere_ as it will simplify cleanup indeed. However, global object might not be the best choice. But the function object of `setInterval` might work better, perhaps (i.e., `function_obj_p` instead of `this_obj`).

One more question occurred to me: when will these properties be removed? Only at cleanup time? Cannot we drop the reference (delete the property) once the event is canceled (in `clearInterval`)?